src/CONST.ts

```diff
@@ -1794,6 +1794,11 @@ const CONST = {
                 XERO: 'xero',
                 NETSUITE: 'netsuite',
             },
+            NAME_MAP: {
```

Comment:
```suggestion
            NAME_USER_FRIENDLY: {
```

This will match the constant names we use in the API and is also more intuitive.

Reply:
Okay updated.